frontend/src/components/notification/notification.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect} from 'react';
+import {Alert, Snackbar} from "@mui/material";
+import CheckIcon from '@mui/icons-material/Check';
+import {useAppDispatch, useAppSelector} from "../../stores/redux/hooks";
+import {selectNotification, showAlert} from "../../stores/redux/notification/notificationSlice";
+import {INotification} from "../../types/notification";
+import {NotificationType} from "../../types/notificationType";
+
+export const createNotification = (message: string): INotification => {
+    return {
+        show: true,
+        type: NotificationType.Success,
+        message: message
+    };
+};
+
+const Notification: React.FC = () => {
+    const [message, setMessage] = React.useState<string>("");
```

Comment:
I don't think you need it

Reply:
without it, i have to use `notification.message`, so when alert is dissappearing, for a few milliseconds alert component appears without message (in general this is not cool)

Reply:
```typescript
const Notification: React.FC = () => {
    const dispatch = useAppDispatch();
    const {show, message, type} = useAppSelector(selectNotification);

    const closeNotification = (event?: React.SyntheticEvent | Event, reason?: string) => {
        if (reason === 'clickaway') {
            return;
        }
        dispatch(showNotification({show: false}));
    };

    return (
        <Snackbar
            open={show}
            autoHideDuration={4000}
            onClose={closeNotification}
            anchorOrigin={{vertical: 'bottom', horizontal: 'center'}}
        >
            <Alert
                variant="filled"
                icon={<CheckIcon fontSize="inherit"/>}
                severity={type}
                onClose={closeNotification}>
                {message}
            </Alert>
        </Snackbar>
    );
};
``` 

then 
```typescript
        showNotification: (state, action: PayloadAction<INotification>) => ({...state, ...action.payload})

```